src/scripts/main.js

```diff
@@ -5,4 +5,47 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
+const table = document.querySelector('.dashboard > tbody');
+
 // write your code here
+for (let i = 0; i < 6; i++) {
```

Comment:
The loop should iterate over all people in the array, not just the first 6. Replace the loop condition with `i < people.length`.